Functions.Templates/Bindings/bindings.json

```diff
@@ -1755,6 +1755,17 @@
             "enabledInTryMode": false,
             "documentation": "$content=Documentation\\Token.md",
             "filters": ["MSGraph"],
+            "AADPermissions": [
```

Comment:
I added it as a special field because it's more like metadata than a setting. Settings are inputs/outputs, whereas this is metadata necessary for the AAD button. 

The permissions do not need to be configurable, as they are relatively constant and would only change if the MS Graph team changed their SDK.